docs/reference/avatar.md

```diff
@@ -0,0 +1,49 @@
+# Avatars
+
+## Project Avatars
+
+This section purpose is to manage the project avatar - the image that is displayed next to your project name in GitLab.
+
+
+The value is a path to an image file relative to the directory where GitLabForm is run. If the value is explicitly set to "", the existing project avatar will be removed.
```

Comment:
To be consistent with how we operate on files in the `files` section, we should allow both absolute and relative paths, but the relative ones should refer to the config file location, not the current working directory, see:

https://github.com/gitlabform/gitlabform/blob/873524007730b883580f7c976577e71ef930ce33/gitlabform/processors/project/files_processor.py#L103-L107